templates/welcome/index.html.twig

```diff
@@ -0,0 +1,32 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}Hello PizzaController!{% endblock %}
+
+{% block body %}
+<style>
```

Comment:
Please remove style from here. This is not the best way to add it, and we will cover styling application later.